test/integration/targets/inventory_foreman_script/runme.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+# OUTPUT_DIR is the build directory
+if [[ ! -d $OUTPUT_DIR ]]; then
+    mkdir -p $OUTPUT_DIR
+fi
+#cd $OUTPUT_DIR
```

Comment:
You shouldn't need to create the output dir since `ansible-test` creates a fresh one for each test run.